### ISSUE TITLE
[Bug]: Arms dealer relocation can be exploited for money

### ISSUE THREAD

**Issue by Alexandre425** [igorkis-scrts/A3-Antistasi-Plus#134]
### Describe the bug

Relocating the arms dealer costs 1000$, but rewards each player with 800$. If there are 2 players or more, the commander can endlessly relocate the arms dealer for a profit.

### How to reproduce

Simply relocate the arms dealer

### Version

2.1.4

### Have you altered the code?

No

### Map

Takistan

### What server?

Private dedicated server

### Mods

```html
CBA_A3 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=450814997
RHSAFRF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=843425103
RHSUSAF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=843577117
RHSSAF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=843632231
ace 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=463939057
RHSGREF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=843593391
Zeus Enhanced 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1779063631
Enhanced Movement 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=333310405
ACE Vehicle Medical 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1911374016
Arsenal Search 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2060770170
Better Inventory 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2791403093
CH View Distance 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=837729515
KAT - Advanced Medical 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2020940806
Ladder Tweak 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1803586009
LAxemann's Environment Ducking 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2812569803
Reduced Haze Mod v3.1 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1397683809
WMO - Walkable Moving Objects 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=925018569
Zeus Enhanced - ACE3 Compatibility 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2018593688
ACE Compat - RHS USAF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=773125288
ACE Compat - RHS AFRF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=773131200
ACE Compat - RHS: SAF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2174495332
ACE Compat - RHS: GREF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=884966711
Weapon Sway 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=784646334
BackpackOnChest - Redux 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2372036642
Antistasi Plus 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2912941775
No Grass Mod 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1482600949
Enhanced Movement Rework 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2034363662
BettIR (Legacy v0.2.1) 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2260572637
Loot to Vehicle for ACE and Antistasi 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1439779114
3CB Factions 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1673456286
CUP Terrains - Core 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=583496184
CUP Terrains - Maps 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=583544987
```

**Comment by FuckingStealingNames** [comment]
I'd personally say it's minuscule profit vs raiding an enemy outpost and selling their goodies.

You'd be going on a waste of time looking at a fast travel screen for a small amount of cash every time.

**Comment by Alexandre425** [comment]
It scales with the amount of players though, and you can turn a 600$ profit every ~30s with just 2 people. It's pretty significant, especially early.

**Comment by igorkis-scrts** [comment]
Addressed in 2.1.7.
